suites/pacific/upgrades/tier-1_upgrade_test-4x-to-5x-container.yaml

```diff
@@ -157,3 +157,8 @@ tests:
         config-file-name: test_multitenant_access.yaml
         timeout: 300
     desc: Perform rgw tests
+
+- test:
+    name: test_cephadm_shell_run.py
```

Comment:
Add polarion id to the suite

**polarion-id: CEPH-83575009**

Reply:
Thanks @Ameenasuhani 
Updated